packages/studio-ui/src/web/components/Layout/TopNav/cloud/pat.ts

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+export const fetchPatStatus = async (pat: string, ac: AbortController): Promise<boolean> => {
+  const resp = await axios.get(`${window.CLOUD_CONTROLLER_ENDPOINT}/v1/pat/authenticate`, {
+    headers: { Authorization: `bearer ${pat}` },
```

Comment:
**question:** isn't it `Bearer ...` and not `bearer ...` or our auth stack doesn't give about case ?

Reply:
it's case insensitive at the [Traefik](https://github.com/botpress/infra/blob/master/cloud/lib/controller-stack.ts#L522) and [CDM](https://github.com/botpress/cdm-cloud/blob/develop/api/src/personal-access-tokens/utils.ts#L12) levels